### PR TITLE
calico version to 1.3.0

### PR DESCRIPTION
**Pull request by DStorck**
fixes https://github.com/samsung-cnct/k2/issues/604

### CODE REVIEW

**Comment by leahnp** [comment]
LGTM

File: ansible/roles/kraken.config/files/gke-config.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ definitions:
             version: v1.4.2
             location: calico/cni
           calicoNode:
-            version: v1.0.0-rc1
+            version: v1.3.0
             location: quay.io/calico/node
           flannel:
             version: v0.6.1
```
Comment (mikeln) [inline ansible/roles/kraken.config/files/gke-config.yaml]: wrong stanza... 
leave this one as it was.
change the `defaultCanalFabric16`

**Comment by mikeln** [comment]
CI fails are seeming all timeouts and flakes

**Comment by mikeln** [comment]
@DStorck you need to rebase this to get it to pass CI

**Comment by mikeln** [comment]
@DStorck you need to rebase this branch for CI to pass

**Comment by leahnp** [comment]
@DStorck does this need to be rebased to pass CI?

**Comment by DStorck** [comment]
@leahnp I have rebased a few times, CI keeps failing. not sure what the deal is.

**Comment by coffeepac** [comment]
calico is up to 2.4.1 now.  they seem to have gone 1.3 -> 2.4.  maybe we should skip this version and jump to 2.4.1

**Comment by mikeln** [comment]
yes.

**Comment by DStorck** [comment]
versions pushed to what @mikeln mentioned above

**Comment by mikeln** [comment]
e2e Error(s) are not related to fabric/network changes. They have to do with pods starting/restarting (was listed as a flake in kubernetes).

**Pull request merged**